(BUG) When running without arguments, errors when parsing for telemetry

**Describe the bug**
When running `pct` without arguments, an error is reported about being unable to index outside of an array.

**To Reproduce**
Steps to reproduce the behavior, run `pct` without any arguments.

**Expected behavior**
Help info is returned.

**Screenshots or logs**
```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/puppetlabs/pdkgo/cmd/root.GetCalledCommand(...)
        cmd/root/root.go:95
main.main()
        main.go:42 +0x5f6
```

**What versions are you running?:**
 - OS: Windows 10
 - PCT Version `0.5.0`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Fixed
+
+- [(GH-285)](https://github.com/puppetlabs/pdkgo/issues/285) Ensure running PCT without arguments does not fail unexpectedly.
+
 ## [0.5.0]
 ### Added
 

--- a/cmd/root/root.go
+++ b/cmd/root/root.go
@@ -92,6 +92,10 @@ func InitConfig() {
 // Idea borrowed from carolynvs/porter:
 // https://github.com/carolynvs/porter/blob/ccca10a63627e328616c1006600153da8411a438/cmd/porter/main.go
 func GetCalledCommand(cmd *cobra.Command) (string, string) {
+	if len(os.Args) < 2 {
+		return "", ""
+	}
+
 	calledCommandStr := os.Args[1]
 
 	// Also figure out the full called command from the CLI

